doc/api/process.md

```diff
@@ -867,7 +871,8 @@ console.log(process.env.foo);
 ```
 
 Assigning a property on `process.env` will implicitly convert the value
-to a string.
+to a string. **This behavior is deprecated.** Future versions of Node.js will
```

Comment:
I’d replace `will` → `may` – at least I, personally, don’t feel comfortable committing to this.